src/main.cpp

```diff
@@ -1,13 +1,17 @@
 #include <Arduino.h>
+#include <IWatchdog.h>
 
 #include "acceleration.hpp"
 #include "barometer.hpp"
 #include "devices.hpp"
 #include "gps.hpp"
 #include "logger.hpp"
+#include "lora.hpp"
 
 constexpr auto baud_rate = 9600l;
-constexpr int interval = 2000;
+constexpr auto logs_interval = 2000;
+constexpr auto lora_interval = 60000;
+constexpr auto watchdog_interval = 10000000;
```

Comment:
10000000 ms? To ponad 2,5 godziny. Jeśli dobrze czytam ten interwał, to wydaje mi się za długi.

Reply:
Biblioteka, argument jest w mikrosekundach. Przy głównym intervale 1s-2s będzie Ok